deployment/proxies.rst

```diff
@@ -137,5 +137,27 @@ these (see ":doc:`/components/http_foundation/trusting_proxies`").
 
 The code for doing this will need to live in your front controller (e.g. ``web/app.php``).
 
+
+My Reverse Proxy do not provide all the standards headers
+---------------------------------------------------------
+For example, as AWS Elastic Load Balancing provide standard header but no X-Forwarded-Host and X-Forwarded, you should use this configuration your front controller::
+   .. code-block:: diff
+
+        // web/app.php
+
+        // ...
+        $request = Request::createFromGlobals();
+        + Request::setTrustedProxies(array('127.0.0.1', $request->server->get('REMOTE_ADDR'))); //be very carefully with this line, see the above chapter "But what if the IP of my Reverse Proxy Changes Constantly!"
+        + 
+        + Request::setTrustedHeaderName(Request::HEADER_CLIENT_HOST, '');// AWS ELB doesn't send X-Forwarded-Host
+        + Request::setTrustedHeaderName(Request::HEADER_FORWARDED, '');// AWS ELB doesn't use RFC 7239
+        + Request::setTrustedProxies(
```

Comment:
According to [this documentation](https://symfony.com/doc/2.7/components/http_foundation/trusting_proxies.html#not-trusting-certain-headers) You're meant to set the trusted header name to `null`. This will work too but it would be better to be consistent.

Reply:
I made this change, but I'm not totally confortable with it since the phpdoc typehint it as a string.
But you're right consistency is probably preferable to type strictness specially since this is only phpdoc not the function signature.